### ISSUE TITLE
Is anyone evaluate successfully on your custom dataset?

### ISSUE THREAD

**Issue by KevinDu1** [traveller59/second.pytorch#358]
I evaluate my custom dataset using 
`python ./pytorch/train.py evaluate --config_path=./configs/car.fhd.config --model_dir=/path/to/model_dir --measure_time=True --batch_size=1
`
After running, I visualize the results but there are little cars are detected, the 3D bboxes  are not in the correct situation in the pointcloud.
My own dataset don't have 3D labels. Does anyone know how I can get the correct results using the pretrained model? 
![image](https://user-images.githubusercontent.com/57181284/78320577-5931b800-759c-11ea-9426-02168e3d41d5.png)

**Comment by simaker** [comment]
which resolution has the used lidar?

**Comment by shiqi1994** [comment]
May I ask how did you evaluate your custom dataset? Thank you very much!
